dev/idm-am-common-user/idm/conf/authentication.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
This whole file should probably be replaced with the stock authentication.json from IDM 6.5, at least until the front-end integration story between AM and IDM is sorted out.